spec/decorators/offence_decorator_spec.rb

```diff
@@ -108,4 +109,48 @@
       it { is_expected.to eql('plea is a string') }
     end
   end
+
+  describe '#mode_of_trial_reason_list' do
+    subject(:call) { decorator.mode_of_trial_reason_list }
+
+    context 'when 1 or more mode of trial reasons exist' do
```

Comment:
There are also edge cases we should consider around
```
 context 'when mode_of_trial_reasons does not contain an expected key' do
```
as i added for pleas.

Given the lack of pact testing and unknowns of common platform generally I think we should test and handle for edges as much as the expected.

Reply:
👍 Have added tests for missing keys ('when it does not contain a code' and 'when it does not contain a description').